tests/HUBViewControllerTests.m

```diff
@@ -270,6 +270,32 @@ - (void)testDelegateNotifiedOfUpdatedViewModel
     XCTAssertEqualObjects(self.viewModelFromDelegateMethod.navigationItem.title, viewModelNavBarTitleB);
 }
 
+- (void)testReloadViewModel
+{
+    NSString * const viewModelNavBarTitleA = @"View model A";
+    NSString * const viewModelNavBarTitleB = @"View model B";
+    
+    self.contentOperation.contentLoadingBlock = ^(id<HUBViewModelBuilder> builder) {
+        builder.navigationBarTitle = viewModelNavBarTitleA;
+        return YES;
+    };
+    
+    [self simulateViewControllerLayoutCycle];
+    
+    XCTAssertEqualObjects(self.viewModelFromDelegateMethod.navigationItem.title, viewModelNavBarTitleA);
+    
+    self.contentOperation.contentLoadingBlock = ^(id<HUBViewModelBuilder> builder) {
+        builder.navigationBarTitle = viewModelNavBarTitleB;
+        return YES;
+    };
+    
+    self.contentReloadPolicy.shouldReload = YES;
```

Comment:
You shouldn't need to set this here. I don't think the reload policy shouldn't be taken into account if the API user explicitly reloads the view controller - it should just reload no matter what.